beets/importer.py

```diff
@@ -1254,8 +1272,9 @@ def user_query(session, task):
         def emitter(task):
             for item in task.items:
                 task = SingletonImportTask(task.toppath, item)
-                task.emit_created(session)
-                yield task
+                tasks = task.emit_created(session)
+                for new_task in tasks:
```

Comment:
For cases like this, I think it would be slightly more readable to collapse these two lines and avoid creating the intermediate, as in `for new task in task.emit_created(...)`.